src/tests/HDFFile.cxx

```diff
@@ -66,6 +66,116 @@ class T_CommandHandler : public testing::Test {
     return event_index[i2] == cue_index;
   }
 
+  static void create_static_file_with_hdf_output_prefix() {
+    using namespace FileWriter;
+    using std::array;
+    using std::vector;
+    using std::string;
+    MainOpt &main_opt = *g_main_opt.load();
+    std::string hdf_output_prefix = "tmp-relative-output";
+    {
+#ifdef _MSC_VER
```

Comment:
Thanks for thinking about running this on Windows, but I don't think it is worth adding untested bits of code like this. I'm going to talk with Freddie about whether we would need to run the writer on Windows at ISIS, because I think we should either not support it at all or have a Windows Jenkins build.